.github/workflows/cypress-test.yml

```diff
@@ -44,8 +67,8 @@ jobs:
         with:
           plugin_name: security-dashboards-plugin
 
-      - name: Configure and Run OpenSearch Dashboards
-        continue-on-error: false
+      # TODO: Switch back to action for checkout after the fix of windows runner.
```

Comment:
Can this TODO statement be removed?

Reply:
Yes we can remove it. I was leaving this comment because we are merging the test into the configuration steps of OSD. That was the work around solution for now, since the checkout is broken on windows runner.